src/Form/organisms/AreaUploadManager.tsx

```diff
@@ -16,9 +17,12 @@ const StyledDropArea = styled(DropArea)`
   width: 100%;
 `;
 
-const InputButtonWrapper = styled.div`
+const InputButtonWrapper = styled.div<{ hasWidth: boolean }>`
   z-index: 99;
   margin-top: 20px;
+  display: flex;
+  justify-content: space-between;
+  width: ${({ hasWidth }) => hasWidth && '285px'};
```

Comment:
I think this can be changed with `gap: 20px` so that we don't have to use prop and fixed width. Please check it.

Reply:
Done